gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -272,6 +278,7 @@
   public static final String DATA_PUBLISHER_FINAL_DIR = DATA_PUBLISHER_PREFIX + ".final.dir";
   public static final String DATA_PUBLISHER_REPLACE_FINAL_DIR = DATA_PUBLISHER_PREFIX + ".replace.final.dir";
   public static final String DATA_PUBLISHER_FINAL_NAME = DATA_PUBLISHER_PREFIX + ".final.name";
+  public static final String DATA_PUBLISHER_GROUP_NAME = DATA_PUBLISHER_PREFIX + ".group.name";
```

Comment:
I don't know. I am not completely sure what these 2 new properties mean: DATASET_URN_KEY and DATA_PUBLISHER_GROUP_NAME. The description of the PR talks about dataset groups which are defined by the dataset.urn property so I assumed a "dataset" and "dataset group" are the same thing.